discord/emoji.py

```diff
@@ -102,13 +58,13 @@ class Emoji(_EmojiTag, AssetMixin):
         "name",
         "_roles",
         "guild_id",
+        "application_id",
```

Comment:
Hi, since it came to light that https://github.com/DisnakeDev/disnake/pull/1209 is a *blatant copy*, I'll share feedback I gave there :)

You should move these two slots to the relevant subclasses, `AppEmoji` doesn't need a `guild_id` slot and vice versa